compiler-core/src/format.rs

```diff
@@ -845,7 +842,11 @@ impl<'comments> Formatter<'comments> {
         }
     }
 
-    fn call<'a>(&mut self, fun: &'a UntypedExpr, args: &'a [CallArg<UntypedExpr>]) -> Document<'a> {
+    fn call<'a>(
+        &mut self,
+        fun: &'a UntypedExpr,
+        args: &'a Vec<CallArg<UntypedExpr>>,
```

Comment:
Why is this `&'a Vec<...>` instead of `&'a [...]` now? It looks like all the pointers to vecs could be slices here.

Reply:
Oops my bad, fixed it!